addons/mail/static/src/core/common/res_users_model.js

```diff
@@ -16,6 +16,21 @@ export class ResUsers extends Record {
     share;
     /** @type {ReturnType<import("@odoo/owl").markup>|string|undefined} */
     signature = fields.Html(undefined);
+    /** @type {string} */
+    email;
+    /** @type {string} */
+    phone;
+    /** @type {string} */
+    im_status;
+    /** @type {number} */
+    company_id;
+
+    static async getAvatarCardData(userId, options = {}) {
```

Comment:
Not necessary to create a separate method when called only once/or just to wrap another method.